mlflow/protos/model_registry.proto

```diff
@@ -654,10 +696,53 @@ message DeleteModelVersionTag {
 
 // Alias for a registered model
 message RegisteredModelAlias {
-  // The name of the registered model associated with the alias.
-  optional string name = 1;
   // The name of the alias.
-  optional string alias = 2;
+  optional string alias = 1;
```

Comment:
Is it safe to reuse an index in proto?

Reply:
Normally I'd say not, but we just merged this PR and there's no place it's being used right now so it should be safe to change.

Reply:
SGTM!